pkg/resource/loki-auth/loki-auth.go

```diff
@@ -46,46 +48,74 @@ func LokiAuthSecretMeta(lc loggedcluster.Interface) metav1.ObjectMeta {
 	return metadata
 }
 
+// listWriteUsers returns a map of users found in a credentialsSecret
+func listWriteUsers(credentialsSecret *v1.Secret) []string {
+	var usersList []string
+	for myUser := range credentialsSecret.Data {
+
+		userTrimmed := strings.TrimSuffix(myUser, "user")
+		// bypass entries that are not a user
+		if userTrimmed == myUser {
```

Comment:
I'd use this instead to remove the continue

```suggestion
		if userTrimmed != myUser && userTrimmed != "read" {
		  usersList = append(usersList, userTrimmed)
		}
```

Reply:
My mind understands better the logic of:
* check for all known exceptions
* then do the job

Also it reduces the number of code imbrication levels.
For these two reasons, I did it this way to make it easier to read.

But that's quite subjective, and you're a more experienced developer than me so I can do it as you suggested.

Reply:
Whether you use continue or not does not really change much but i'd probably have only 1 if as it's not really nested but i'm definitely not blocking